### PR TITLE
Changed Recipe for Planar Conduit and Transduction Amplifier

### PR DESCRIPTION
**Pull request by Azemari**
**Old Recipes:**
Transduction Amplifier
image
Planar Conduit
image

**New Recipes:**
Transduction Amplifier
image
Planar Conduit
image

### CODE REVIEW

**Comment by Azemari** [comment]
I'm not sure how to link issues: [https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7222#issue-777489590](url)

File: scripts/Thaumic-Horizons.zs [hunk 1]
```diff
@@ -26,19 +26,19 @@ mods.thaumcraft.Infusion.removeRecipe(<ThaumicHorizons:recombinator>);
 
 
 // --- Planar Conduit
-mods.thaumcraft.Arcane.addShaped("planarTheory", <ThaumicHorizons:planarConduit>, "aer 100, aqua 100, ignis 100, terra 100, ordo 100, perditio 100", [
-[<ore:plateShadow>, <ore:plateEnderEye>, <ore:plateShadow>],
+mods.thaumcraft.Arcane.addShaped("planarTheory", <ThaumicHorizons:planarConduit>, "aer 50, aqua 50, ignis 50, terra 50, ordo 50, perditio 50", [
```
Comment (Dream-Master) [inline scripts/Thaumic-Horizons.zs:29]: no one said to change that to 50

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: scripts/Thaumic-Horizons.zs (hunk 1 above)
Comment (KiloJoel) [reply scripts/Thaumic-Horizons.zs:29]: I did in the issue, I realised later.

Comment (Dream-Master) [reply scripts/Thaumic-Horizons.zs:29]: Wait i guess the primal pearl in the middle need more than 50 vis so why we change that to 50 ?

Comment (KiloJoel) [reply scripts/Thaumic-Horizons.zs:29]: Primal charm is 50. If the planar conduit costs 100 vis, you need an augmented greatwood scepter (vacuum freezer), or high level caps (forbidden research or titanium)
image

Comment (Azemari) [reply scripts/Thaumic-Horizons.zs:29]: What about 75? Then you need an iron capped greatwood scepter at least?

Comment (Dream-Master) [reply scripts/Thaumic-Horizons.zs:29]: yea make it 75

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: scripts/Thaumic-Horizons.zs [hunk 2]
```diff
@@ -26,19 +26,19 @@ mods.thaumcraft.Infusion.removeRecipe(<ThaumicHorizons:recombinator>);
...
-[<ore:plateShadow>, <ore:plateEnderEye>, <ore:plateShadow>]]);
+[<ore:plateVoid>, <ore:plateEnderEye>, <ore:plateVoid>]]);
 // -
 mods.thaumcraft.Research.setAspects("planarTheory", "vacuos 18, praecantatio 15, alienis 12, auram 9, perditio 6, tenebae 3");
 mods.thaumcraft.Research.setComplexity("planarTheory", 4);
 mods.thaumcraft.Warp.addToResearch("planarTheory", 1);
 
 // --- Transduction Amplifier
-mods.thaumcraft.Arcane.addShaped("transductionAmplifier", <ThaumicHorizons:transductionAmplifier>, "aer 150, ignis 150, ordo 150", [
+mods.thaumcraft.Arcane.addShaped("transductionAmplifier", <ThaumicHorizons:transductionAmplifier>, "aer 50, ignis 50, ordo 50, terra 50, aqua 50, perditio 50", [
```
[the remark this answers is not in the document]
Comment (KiloJoel) [reply scripts/Thaumic-Horizons.zs:39]: 75 would be ok. A bit high for what it is. But not excessively so.

[the remark this answers is not in the document]
Comment (Dream-Master) [reply scripts/Thaumic-Horizons.zs]: ok @azemari Change it to 75 please

[the remark this answers is not in the document]
Comment (Azemari) [reply scripts/Thaumic-Horizons.zs]: I have done in changeset 4363d8b

**Review by Dream-Master — APPROVED** [review]

File: scripts/Thaumic-Horizons.zs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Dream-Master) [reply scripts/Thaumic-Horizons.zs]: ok

**Pull request merged**